sync/src/block/extension.rs

```diff
@@ -479,6 +479,7 @@ impl Extension {
                 _ => {}
             }
         }
+        // I think it already had to be removed from drain() in line 468
```

Comment:
No. `remove_target` removes the downloading list also. But `drain` removes the downloaded list only.

Reply:
By the way, I think this change itself looks good.

Reply:
Would you cleanup the PR and remove the WIP tag?

Reply:
Okay

Reply:
`exists` were removed from downloading list in `import_bodies()`